docker_mcp/core/transfer/archive.py

```
@@ -201,56 +201,66 @@ async def create_archive(
+
+                # Create timestamped archive name
+                timestamp = datetime.now().strftime("%Y%m%d_%H%M%S")
+                archive_file = f"{temp_dir}/{archive_name}_{timestamp}.tar.gz"
+
+                # Find common parent and convert to relative paths
+                common_parent, relative_paths = self._find_common_parent(volume_paths)
+
+                # Build tar command with -C to change directory
+                import shlex
+
+                tar_cmd = ["tar", "czf", archive_file, "-C", common_parent] + exclude_flags + relative_paths
```

Ruff (0.14.4)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)